sdk/appconfiguration/azure-appconfiguration/tests/test_azure_configuration_client.py

```diff
@@ -464,3 +465,40 @@ def test_set_read_only(self, client, appconfiguration_connection_string, test_co
         with pytest.raises(ResourceModifiedError):
             client.set_read_only(set_kv, True, match_condition=MatchConditions.IfNotModified)
 
+    @app_config_decorator
+    def test_sync_tokens(self, client):
+
+        sync_tokens = copy.deepcopy(client._sync_token_policy._sync_tokens)
+        keys = list(sync_tokens.keys())
+        seq_num = sync_tokens[keys[0]].sequence_number
```

Comment:
I am afraid it is guaranteed there is only one key in sync_tokens.keys() and the updated one must be sync_tokens[keys[0]].